MetaMask/metamask-mobile
# feat: 1957 crash screen redesign

## **Description**



Refactors ErrorBoundary to match [updated designs in Figma](https://www.figma.com/design/DM5G1pyp74sMJwyKbw1KiR/Error-message-and-bug-report?node-id=1-5473&node-type=section&t=98CWfEC4JrKwpNCE-0)

## **Related issues**

Fixes: [#1957](https://github.com/MetaMask/mobile-planning/issues/1957)

## **Manual testing steps**

1. Go WalletView
2. Add:
    ```
    useEffect(() => {
      throw new Error('Test Error');
    }, []);
    ```
3. Restart the app
4. Dismiss the Red Screen
5. See the new Error Screen

## TESTING THE BUILD:
1. Create a QA build in Bitrise
2. force an error
3. report the error
4. check [Sentry](https://metamask.sentry.io/feedback/?feedbackSlug=test-metamask-mobile%3A6031686409&mailbox=ignored&project=2651591&referrer=feedback_list_page&statsPeriod=30d)
5. Should see user feedback in the Sentry report
6. Example:
![image](https://github.com/user-attachments/assets/c1737758-5739-48f9-96d1-798e523f5be3)


## **Screenshots/Recordings**



### **Before**


<img width="753" alt="image" src="https://github.com/user-attachments/assets/136542c2-1578-4e87-9922-154f21a70c4e">


### **After**




https://github.com/user-attachments/assets/7dbe5e04-0152-41bc-984a-37cba98c4eeb


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
